SIL.Scripture.Tests/BCVRefTests.cs

```diff
@@ -553,6 +553,117 @@ public string MakeReferenceString_DifferentBooks(string cvSeparator, string brid
 				new BCVRef(43, 3, 16), cvSeparator, bridge, true);
 		}
 
+		/// ------------------------------------------------------------------------------------
+		/// <summary>
+		/// Tests the EqualityOperator when both references are null
+		/// </summary>
+		/// ------------------------------------------------------------------------------------
+		[Test]
+		public void EqualityOperator_BothNull()
+		{
+			// ReSharper disable once EqualExpressionComparison
+			// ReSharper disable once RedundantCast
+			Assert.IsTrue((BCVRef)null == (BCVRef)null);
+		}
+
+		/// ------------------------------------------------------------------------------------
+		/// <summary>
+		/// Tests the EqualityOperator when the first reference is null
+		/// </summary>
+		/// ------------------------------------------------------------------------------------
+		[Test]
+		public void EqualityOperator_LeftReferenceNull()
+		{
+			Assert.IsFalse(null == new BCVRef(3, 4, 5));
+		}
+
+		/// ------------------------------------------------------------------------------------
+		/// <summary>
+		/// Tests the EqualityOperator when the second reference is null
+		/// </summary>
+		/// ------------------------------------------------------------------------------------
+		[Test]
+		public void EqualityOperator_RightReferenceNull()
+		{
+			Assert.IsFalse(new BCVRef(5, 2, 1) == null);
+		}
+
+		/// ------------------------------------------------------------------------------------
+		/// <summary>
+		/// Tests the EqualityOperator when the first reference is invalid
+		/// </summary>
+		/// ------------------------------------------------------------------------------------
+		[TestCase(-1, 2, 3)]
+		[TestCase(1, 3, 2)]
```

Comment:
Why is this an invalid reference? (similar in the other tests below)

Reply:
It wasn't. That was me working too fast on a sleep-deprived brain. Fixed it now.